_rules/word-spacing-not-important-9e45ec.md

```diff
@@ -157,6 +161,65 @@ The [computed][] value of the `word-spacing` property of this `p` element is **n
 </p>
 ```
 
+#### Passed Example 9
+
+This `p` element only has one text node descendant, and it is [wide enough][] because its parent is the `span` element, with a `word-spacing` of `2px`, more than `0.16` times the `font-size` of `10px`.
+
+```html
+<p style="font-size: 24px; word-spacing: 2px !important">
+	<span style="font-size: 10px;">
+		The toy brought back fond memories of being lost in the rain forest.
+	</span>
+</p>
+```
+
+#### Passed Example 10
+
+This `p` element has no text node descendant and therefore matches the [spaced text](#9e45ec:spaced-text) condition.
+
+```html
+<p style="letter-spacing: 1px !important; border-top: 1px solid black;">
+	<!-- empty div, border to make it "visible" -->
+</p>
+```
+
+#### Passed Example 11
+
+This `p` element only has one text node descendant, and it is [empty](#9e45ec:empty).
+
+```html
+<p style="word-spacing: 1px !important; border-top: 1px solid black;">
+	&nbsp:<!-- empty div, border to make it "visible" -->
+</p>
+```
+
+#### Passed Example 12
+
+Both this `p` and `span` elements have a single text node descendant, whose parent (the `span` element) is [wide enough][].
```

Comment:
Same here... for the `p` element, what is relevant is the cascade condition, right?